spec/outputs/sns_spec.rb

```diff
@@ -2,17 +2,110 @@
 require "logstash/devutils/rspec/spec_helper"
 require 'logstash/outputs/sns'
 require 'logstash/event'
+require "logstash/plugin_mixins/aws_config"
+
+require "aws-sdk" # TODO: Why is this not automatically brought in by the aws_config plugin?
```

Comment:
Ah I needed that there for tests to run, since some of my tests don't call `#register`